In-house patch to remove unnecessary dependencies from Horizon's
requirements files. The specific reasons are as follows:

eventlet	Not applicable

kombu		Not applicable

pyscss		Not applicable

cinderclient	push to version 1.3.1

--- horizon-2015.1.2/horizon.egg-info/requires.txt.orig	2015-10-13 11:09:26.000000000 -0600
+++ horizon-2015.1.2/horizon.egg-info/requires.txt	2015-11-19 15:15:17.976212795 -0700
@@ -5,19 +5,16 @@ Pint>=0.5 # BSD
 django-compressor>=1.4
 django-openstack-auth<1.3.0,>=1.2.0
 django-pyscss<2.0.0,>=1.0.3 # BSD License2 clause
-eventlet!=0.17.0,>=0.16.1
 httplib2>=0.7.5
 iso8601>=0.1.9
-kombu>=2.5.0
 netaddr>=0.7.12
 oslo.concurrency<1.9.0,>=1.8.2 # Apache-2.0
 oslo.config<1.10.0,>=1.9.3 # Apache-2.0
 oslo.i18n<1.6.0,>=1.5.0 # Apache-2.0
 oslo.serialization<1.5.0,>=1.4.0 # Apache-2.0
 oslo.utils!=1.4.1,<1.5.0,>=1.4.0 # Apache-2.0
-pyScss<1.3,>=1.2.1 # MIT License
 python-ceilometerclient<1.2,>=1.1.1
-python-cinderclient<1.2.0,>=1.1.0
+python-cinderclient<=1.3.1,>=1.1.0
 python-glanceclient<0.18.0,>=0.15.0
 python-heatclient<0.5.0,>=0.3.0
 python-keystoneclient<1.4.0,>=1.2.0

--- horizon-2015.1.2/requirements.txt.orig	2015-10-13 11:07:24.000000000 -0600
+++ horizon-2015.1.2/requirements.txt	2015-11-19 15:16:14.438547386 -0700
@@ -15,19 +15,16 @@ Pint>=0.5 # BSD
 django-compressor>=1.4
 django-openstack-auth<1.3.0,>=1.2.0
 django-pyscss<2.0.0,>=1.0.3 # BSD License (2 clause)
-eventlet!=0.17.0,>=0.16.1
 httplib2>=0.7.5
 iso8601>=0.1.9
-kombu>=2.5.0
 netaddr>=0.7.12
 oslo.concurrency<1.9.0,>=1.8.2 # Apache-2.0
 oslo.config<1.10.0,>=1.9.3 # Apache-2.0
 oslo.i18n<1.6.0,>=1.5.0 # Apache-2.0
 oslo.serialization<1.5.0,>=1.4.0 # Apache-2.0
 oslo.utils!=1.4.1,<1.5.0,>=1.4.0 # Apache-2.0
-pyScss<1.3,>=1.2.1 # MIT License
 python-ceilometerclient<1.2,>=1.1.1
-python-cinderclient<1.2.0,>=1.1.0
+python-cinderclient<=1.3.1,>=1.1.0
 python-glanceclient<0.18.0,>=0.15.0
 python-heatclient<0.5.0,>=0.3.0
 python-keystoneclient<1.4.0,>=1.2.0
